hack/test-end-to-end-docker.sh

```diff
@@ -15,9 +15,8 @@ echo "[INFO] Starting containerized end-to-end test"
 
 unset KUBECONFIG
 
-TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-e2e-containerized}"
 setup_env_vars
+setup_temp_dir_vars "test/end-to-end/containerized/"
```

Comment:
I see. Good to know, thanks for sharing! These scripts run our test suites, so I think we're safe.